test/test_p4est_2d.jl

```diff
@@ -122,6 +122,13 @@ isdir(outdir) && rm(outdir, recursive=true)
               0.0016290067532561904],
       tspan = (0.0, 0.02))
   end
+
+  @trixi_testset "elixir_euler_forward_step.jl" begin
```

Comment:
If you haven't done it yet, please check the runtime for this test just to be sure.

Reply:
I can move it to be with the Euler. Out of habit I just stuck the new test at the end of the file. I am playing with the mesh and some of the parameters right now to try and remove the fictitious boundary layer caused by the corner resolution. So the test will get changed anyway

Reply:
Yes, I prefer keeping all Euler tests together. Moreover, it would be great to reduce CI pressure, if reasonably possible

Reply:
Done. I moved the test values in the file and also shorted it. Now it only takes 15 timesteps. I will check the CI time...

Reply:
I shrank the final time to decrease the number of time steps but I think it still takes 209s (or approx. 3 minutes) to run the new test. I am not sure what else to do to shorten it...

Reply:
What about skipping the coverage test for this elixir? Without coverage, it's just 25 seconds.

Reply:
I am okay with that as long as others are. What do you think @sloede @jlchan @gregorgassner ? This might drop coverage slightly because of the new initial condition routine and only this example uses the outflow BC.

Reply:
> Question: is this really the only example using outflow BC?

The double Mach elixir that I made today also uses it. That will go into a different PR but I think it would have a faster testing time because it does not have such extreme refinement that this test example requires for the corner

Reply:
Then I would prefer skipping coverage here. You can go ahead, we merge this, and decide where to put the BC when you make your new PR. Does that sound good to you?

Reply:
Sounds good to me

Reply:
I merged `main` with new keyword argument `skip_coverage` of `@test_trixi_include` into your PR, @andrewwinters5000